site/layouts/default.hbs

```diff
@@ -31,10 +36,23 @@
 			{{>contentbuttons}}
 			{{>body}}
 {{#if page.deptfeature}}
-	{{>deptfeatures}}
+	{{#is page.pageType "theme-topic"}}{{>deptfeatures-wet-126}}{{/is}}
+	{{#isnt page.pageType "theme-topic"}}{{>deptfeatures}}{{/isnt}}
+{{/if}}
+{{#if page.contributors}}
+	<div class="container">
+	{{>contributors}}
```

Comment:
Add this include in the page theme-topic template, not from the main layout because it won't be reused on all other page template